src/commons/application/ApplicationTypes.ts

```diff
@@ -207,26 +209,29 @@ export const styliseSublanguage = (chapter: Chapter, variant: Variant = Variant.
   }
 };
 
-export const sourceSublanguages: Language[] = [
-  { chapter: Chapter.SOURCE_1, variant: Variant.DEFAULT },
+export const sourceSublanguages = [
```

Comment:
Would be good to type this anyway, if only to prevent typos.

Reply:
This would be caught when the declaration of `ALL_LANGUAGES` is processed, but it still would be good to catch any such issues earlier.

Reply:
We can't type it as `Language[]` because the flags/other config stuff like `usesSubstVisualizer` are not defined in the `Language` interface. We can't type as `SALanguage[]` either since they are missing keys like `mainLanguage`.

Any errors will actually be caught earlier, not in `ALL_LANGUAGES`, but in `sourceLanguages`/`pyLanguages`/`schemeLanguages` as they are all typed as `SALanguage[]`.

Since errors would be caught anyway, moreover, these `xxxSubLanguages` are not exported (only the languages are), I think it's fine to leave them untyped.

What do you think?

Reply:
I was thinking that it could have its own type which is a subset of `SALanguage`, but you're right that the errors will be caught anyway so this is more of a minor nit.